pkgs/top-level/python-packages.nix

```diff
@@ -169,6 +169,26 @@ in {
     };
   };
 
+  antfs-cli = buildPythonApplication rec {
```

Comment:
Since this is an application, it should go into the directory structure similarly to garmin-uploader.